tools/snpsift_filter_builder/.shed.yml

```diff
@@ -0,0 +1,14 @@
+# .shed.yml supporting automatic pushes.
+owner: artbio
+name: snpsift_filter_builder
+description:  snpsift_filter_builder creates the filter line parameter to filter variants with the SnpSift tool
+long_description: |
+    Takes a table describing samples (normal/patients) and a VCF file to create the command line for filtering in SnpSift tool
```

Comment:
Takes a table describing samples (normal/patients) and a VCF file to create a command line for filtering variants `using the` SnpSift tool.